src/platform/user/profile/vap-svc/containers/CopyMailingAddress.jsx

```diff
@@ -51,6 +51,9 @@ class CopyMailingAddress extends React.Component {
     // If mailing + home addresses are the same, clear the home address
     if (this.areHomeMailingAddressesEqual()) {
       const clearedHomeAddress = mapValues(mailingAddress, () => null);
+
+      // We need the id to remain the same to prevent POST calls
+      clearedHomeAddress.id = mailingAddress.id || null;
```

Comment:
can you also set the country to USA for the cleared address? `countryCodeIso3 = USA.COUNTRY_ISO3_CODE`